Sources/App/githubRoutes.swift

```diff
@@ -0,0 +1,59 @@
+import Vapor
+
+public func githubRoutes(router: GithubCommandRouter) throws {
```

Comment:
Bot commands can be registered as "routes", just like you would with http routes. This setup should help keep things modular and somewhat organized.